src/components/PipelineRun/PipelineRunListView/PipelineRunListRow.tsx

```diff
@@ -115,6 +131,23 @@ const BasePipelineRunListRow: React.FC<React.PropsWithChildren<BasePipelineRunLi
           '-'
         )}
       </TableData>
+      <TableData className={pipelineRunTableColumnClasses.triggeredBy}>
+        {gitProvider === 'github' ? (
+          <GithubIcon className="git-icon" />
+        ) : gitProvider === 'gitlab' ? (
+          <GitlabIcon className="git-icon" />
+        ) : null}
+
+        {eventType ? (
+          <ExternalLink
+            href={`https://${gitProvider}.com/${repoOrg}/${repoURL}#${prNumber}`}
+            text={EventTypeLabel[eventType] ?? '-'}
+            hideIcon={true}
```

Comment:
if it's an external link dont hide the icon, only hide it if it's a redirect in the konflux-ui